docs/general/create-mf-app.mdx

```diff
@@ -222,6 +222,28 @@ module.exports = withZephyr()({
 
 ```
 
+### Zephyr Dependencies in package.json optional
+
+For example we have [react-tractor-sample](https://github.com/ZephyrCloudIO/zephyr-examples/tree/main/examples/react-tractor-sample).
```

Comment:
Chat GPT is very good to rewrite text into one more suitable for docs :)

https://chatgpt.com/share/6769629a-7764-8001-a9b7-f887fa31d6a4

---

For example, consider the [react-tractor-sample](https://github.com/ZephyrCloudIO/zephyr-examples/tree/main/examples/react-tractor-sample) repository. It includes an application called `team-red`, which depends on two other applications: `team-green` and `team-blue`.

To specify the version of each dependency, you can define `zephyrDependencies` in the `package.json` file like this:

```json title=package.json
{
  "name": "team-red",
  "version": "0.1.0",
  "scripts": {
    "build": "nx run team-red:build --skip-nx-cache"
  },
  "private": true,
  "zephyrDependencies": {
    "team-blue": "0.1.0",
    "team-green": "0.1.0"
  }
}
```

If you omit the `zephyrDependencies` configuration, the latest versions of the remote applications will be used by default.